src/Scry.ts

```diff
@@ -54,19 +54,21 @@ async function queryApi<T> (
 
 	let err: SearchError | undefined;
 
-	const result = await request({
-		body: post,
-		json: true,
-		method: post ? "POST" : "GET",
-		qs: query,
-		uri: `${endpoint}/${apiPath}`,
-	}).catch(({ response }) => {
-		err = response.body;
+	const path = Array.isArray(apiPath)
+		? (apiPath = apiPath.join("/"))
+		: apiPath.toString();
+	const url = `${endpoint}/${path}`;
+
+	const result = await (post
+		? Axios.post(url, post, { params: query })
+		: Axios.get(url, { params: query })
```

Comment:
Use the plain function call so that it's 1:1 to the request code https://www.npmjs.com/package/axios#axiosconfig